integration/tests/common/blockstate/block_state.go

```diff
@@ -1,4 +1,4 @@
-package common
+package blockstate
```

Comment:
Moving `blockstate` to a separate subpackage to deter other trackers (e.g., `receiptstate`) from accessing its unexported fields without synchronization, and hence resulting in a race condition.